js/components/Verse.react.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Verse
+ */
+
+import { asyncEnableRecall, asyncDisableRecall } from '../actions/AppActions';
+
+import React, { Component } from 'react';
+
+class Verse extends Component {
+  render() {
+    const { index, book, chapter, text, verse, isRecalling, dispatch } = this.props;
+
+    function splitText(text) {
+      return text.split(' ').map(function (word) {
+        return word[0] + word.slice(1, word.length).replace(/\w/g, ' ');
+      }).join(' ');
+    }
+
+    return (
+      <li>
+        <div className="passage-card">
+          <p className="passage-metadata">{ book + ' ' + chapter + ':' + verse }</p>
```

Comment:
I was basing this on the context of always having each verse rendered separately in it's own card, regardless of what overarching mode (single, multi, selective) you were in. I guess we hadn't clearly defined if that was our goal based on the requirement (perhaps better call it desire) to aid in specific verse recall.

As far as grouping of selective passages (whether pacing or otherwise), I would imagine that mode sitting beside single and multi, and containing the range of verses to render. That said, we'll probably just want to change the passage reducer in the future to render only verses we tell it, instead of storing all of the verses and returning all or one based on index. Saying that based on scalability in the future.